discovery-provider/src/queries/get_unclaimed.py

```diff
@@ -0,0 +1,48 @@
+from random import randrange
+
+from src import exceptions
+from src.models.playlists.aggregate_playlist import AggregatePlaylist
+from src.models.tracks.aggregate_track import AggregateTrack
+from src.models.users.aggregate_user import AggregateUser
+from src.utils.db_session import get_db_read_replica
+
+MAX_USER_ID = 999999999  # max for reward specifier id
+MAX_POSTGRES_ID = 2147483647
+
+
+def get_unclaimed(type):
+
+    if type not in ["track", "playlist", "user"]:
+        raise exceptions.ArgumentError(
+            "Invalid type provided, must be one of 'track', 'playlist', 'user'"
+        )
+
+    db = get_db_read_replica()
+    with db.scoped_session() as session:
+        for _ in range(10):
+            is_claimed = True
+            random_id = randrange(MAX_POSTGRES_ID)
```

Comment:
There is a minimum val for user, track and playlist that I believe we have in discovery as an env